schema-v0.0.0.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://fair.io/schema-v0.0.0.json",
+  "title": "Annual OSS Pledge Report",
+  "description": "A company's annual OSS Pledge self-report",
+  "type": "object",
+  "properties": {
+    "dateYearEnding": {
+      "description": "The end date for the year in question",
+      "type": "string",
+      "format": "date"
+    },
```

Comment:
Maybe it would be good to split this up into `reportingPeriodStart` and `reportingPeriodEnd`. It seems like it might be desirable in certain scenarios to have reporting on a different basis than yearly. For example, after a company makes the pledge, we have no information about any of their contributions for a whole year, which seems like a long time. Other than that, maybe we want to offer companies the flexibility to report non-yearly if they have this need.

Reply:
This seems like over-engineering to me. Nearly every company has an annual budget cycle. I think we should keep the system simple to ease both implementation and comprehension when comparing across companies.